CONTRIBUTING-it.md

```diff
@@ -0,0 +1,216 @@
+*Leggilo in altre lingue: [English](CONTRIBUTING.md), [Français](CONTRIBUTING-fr.md), [Español](CONTRIBUTING-es.md), [简体中文](CONTRIBUTING-zh.md), [繁體中文](CONTRIBUTING-zh_TW.md), [فارسی](CONTRIBUTING-fa_IR.md), [Tiếng Việt](CONTRIBUTING-vn.md), [Русский](CONTRIBUTING-ru.md).*
```

Comment:
:information_source: An oportunity to link other translations not merged yet :tada: 

- #5720 German